wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp

```diff
@@ -197,7 +197,7 @@ void CommandScheduler::Run() {
     if constexpr (frc::RobotBase::IsSimulation()) {
       subsystem.getFirst()->SimulationPeriodic();
     }
-    m_watchdog.AddEpoch("Subsystem Periodic()");
+    m_watchdog.AddEpoch(GetTypeName(subsystem) + ".Periodic()");
```

Comment:
I'm not a fan of calling GetTypeName() this frequently, as it can be expensive.  Instead, build this string during AddSubsystem() and just use it here.

Reply:
That's what's done with commands, and I'm not sure where it would make sense to store the strings.

Reply:
In the m_subsystems map, you'll need to store more than just the subsystem pointer.

Reply:
Now that I think of it, I think that `Subsystem` should have a `GetName` method--sometimes teams create two of the same subsystem class intentionally (swerve modules, etc.). It would also be better for Sendable.